addon/components/columns/base.js

```diff
@@ -73,6 +73,21 @@ const Column = Component.extend(DraggableColumnMixin, {
   sortIcons: null,
 
   /**
+   * @property sortIcon
+   * @type {String}
+   */
+  sortIcon: computed('column.sorted', 'column.ascending', function() {
+    let sorted = this.get('column.sorted');
+
+    if (sorted) {
+      let ascending = this.get('column.ascending');
+      return ascending ? 'iconAscending' : 'iconDescending';
+    }
+
+    return 'iconSortable';
+  }),
```

Comment:
Sounds good.
The only thing I can add is a :bike: :house_with_garden: on the name -- `sortIconProperty` or `sortIconAttribute` just so it's clear that this string represents the attribute that will be pulled from to generate the icon simply from the name (we could add docs for it too)